config/sync/views.view.discover_topics.yml

```diff
@@ -0,0 +1,196 @@
+uuid: 37dffbcd-38e1-4fcf-8802-8ad3041f81d3
+langcode: en
+status: true
+dependencies:
+  config:
+    - entityqueue.entity_queue.discover_topics
+  module:
+    - entityqueue
+id: discover_topics
+label: 'Discover topics'
+module: views
+description: ''
+tag: ''
+base_table: entity_subqueue_field_data
```

Comment:
FYI, indeed the views were very limitted. Up to now, we were indexing in the SOLR endpoint whatever we needed in order to create a view on top of the search_api endpoint. The way we would go with this would be to create a computed field with the entity queues the entity would belong and index this information. But anyway, I think it is a more performant solution to use the block.

Reply:
We have the `EntityQueueBlock` now which will just retrieve the referenced entities directly from the `EntityQueue` entity and render them. Still I don't think we need to change this now since it works and it looks like we can display them as designed, even though the HTML is not semantically correct. If the frontend team objects when they start on this we can still change it.